### PR TITLE
doc: Samples for different storages （AlluxioRuntime）

### LINKED ISSUE #797: [DOC] Samples for different storages （AlluxioRuntime）
**Provide a link to that doc page:**

Add doc on how to support different storage.

S3
HDFS
Ceph S3
PV
Minio

image

Maybe we can refer https://kubernetes.io/docs/concepts/storage/persistent-volumes/
https://docs.alluxio.io/os/user/stable/en/ufs/Ceph.html

**What is the defect and your suggestions on improvement:**

### PR DESCRIPTION
**Pull request by baowj-678**
Signed-off-by: baowj <bwj_678@qq.com>

### Ⅰ. Describe what this PR does

Add doc on how to support different storage.

### Ⅱ. Does this pull request fix one issue?

fixes #797

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: docs/en/samples/accelerate_different_storage.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+**Create Dataset Resource Object**
+
+``` yaml
+$ cat << EOF > dataset.yaml
+apiVersion: data.fluid.io/v1alpha1
+kind: Dataset
+metadata:
+  name: my-hdfs
+spec:
+  mounts:
```
Comment (allenhaozi) [inline docs/en/samples/accelerate_different_storage.md:33]: Can you add more mountpoints? Is subpath supported?

File: docs/en/samples/accelerate_different_storage.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                  name: mysecret
+                  key: aws.accessKeyId
+            - name: aws.secretKey
+              valueFrom:
+                secretKeyRef:
+                  name: mysecret
+                  key: aws.secretKey
+    ```
+
+* PVC：`- mountPoint: pvc://nfs-imagenet`；
```
Comment (allenhaozi) [inline docs/en/samples/accelerate_different_storage.md:72]: I suggest here is the complete dataset example

File: docs/en/samples/accelerate_different_storage.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+      name: hdfs
+EOF
+```
+
+```
+$ kubectl create -f dataset.yaml
+```
+
+Here, we will create a resource object whose *kind* is `Dataset` , Dataset is a Custom Resource Definition (CRD) defined by Fluid. The CRD is used to tell Fluid where to find the data you need.
+
```
[the remark this answers is not in the document]
Comment (baowj-678) [reply docs/en/samples/accelerate_different_storage.md:44]: OK, I'll delete it later.

File: docs/en/samples/accelerate_different_storage.md (hunk 1 above)
Comment (baowj-678) [reply docs/en/samples/accelerate_different_storage.md:33]: OK, I can add more mountpoints.

File: docs/en/samples/accelerate_different_storage.md (hunk 2 above)
Comment (baowj-678) [reply docs/en/samples/accelerate_different_storage.md:72]: OK, I'll put complete dataset example here.

**Review by allenhaozi — APPROVED** [review]
LGTM

**Review by cheyang — APPROVED** [review]

**Pull request merged**